web/styles/right_sidebar.css

```diff
@@ -484,13 +484,33 @@ $user_status_emoji_width: 24px;
     }
 }
 
-.right-sidebar-shortcuts {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
+.invite-user-shortcut {
     /* The margin top is calculated from a legacy 25px height,
        from a 20px line of text and 5px of margin top. We calculate
        a scaling margin-top by subtracting the em-unit line height
        from the legacy value. */
     margin-top: calc(25px - (var(--legacy-body-line-height-unitless) * 1em));
+    margin-right: var(--width-simplebar-scroll-hover);
+    padding-left: var(--right-sidebar-left-margin);
+    border-radius: 4px;
+
+    &:hover {
+        background: var(--color-background-sidebar-action-heading-hover);
+        box-shadow: inset 0 0 0 1px var(--color-shadow-sidebar-row-hover);
+    }
+
+    .invite-user-link {
+        display: flex;
```

Comment:
I wonder if this shouldn't be gridded like the other rows, and perhaps make also use of the `--line-height-sidebar-row` to establish a track to put these on, rather than relying on element heights in flexbox?

Reply:
I'm open to that, though I'm not quite sure what that would look like.

Reply:
I'm also curious to understand more about why grid is preferred here over flexbox.

Reply:
In this case, it's because we can use grid to specify the height of a row, and then use alignment properties to center it (or whatever) within the row. With flexbox, you have to tinker with the dimensions of individual flex items. While you can set a `height` value on a flex container, I think grid is easier to reason about.